dom-manipulation/main.js

```diff
@@ -0,0 +1,25 @@
+var counter = 1;
+
+var hotButton = document.querySelector('.hot-button');
+
+var clickcount = document.querySelector('.click-count');
+
+// hotButton.addEventListener('click', function () { counter++; console.log(counter); });
+
+hotButton.addEventListener('click', function () {
+  clickcount.textContent = 'Clicks: ' + counter++;
```

Comment:
This will make your click count not update the first time you click it, and your count will always be one behind the actual count.